pkg/serverless/invocationlifecycle/lifecycle.go

```diff
@@ -229,6 +229,7 @@ func (lp *LifecycleProcessor) OnInvokeEnd(endDetails *InvocationEndDetails) {
 					lp.requestHandler.inferredSpans[1].CompleteInferredSpan(lp.ProcessTrace, lp.getInferredSpanStart(), endDetails.IsError, lp.GetExecutionInfo().TraceID, lp.GetExecutionInfo().SamplingPriority)
 					log.Debug("[lifecycle] The secondary inferred span attributes are %v", lp.requestHandler.inferredSpans[1])
 				}
+				lp.GetInferredSpan().AddTagToInferredSpan("http.status_code", statusCode)
```

Comment:
Is `statusCode` guaranteed to exist?

Reply:
statusCode is a string and in go it defaults to an empty string if not initialized

<img width="1512" alt="Screen Shot 2023-04-27 at 1 07 35 PM" src="https://user-images.githubusercontent.com/49878080/234979359-70bbf2f6-f41e-40e5-9568-782f77242b9f.png">

<img width="952" alt="Screen Shot 2023-04-27 at 1 12 56 PM" src="https://user-images.githubusercontent.com/49878080/234980280-8a98aa1c-fd03-4ec6-be56-e17a82f3642e.png">